crypto/hash/hashfuncs.go

```diff
@@ -22,6 +22,11 @@ import (
 	"hash/fnv"
 )
 
+type HashSuite struct {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type HashSuite should have comment or be unexported